src/pages/meta-tag.js

```diff
@@ -32,74 +33,66 @@ export default function MetaTag(props) {
+            onClick={() => {
+              setInvalidUrl(false)
+              setPointer(parsePointerUrl(pointerInput))
+            }}
+          >
+            Generate
+          </button>
+        </form>
+        <p>
+          Read our <Link to='/docs'>docs</Link> to learn more about Web
+          Monetization. If you're interested in splitting revenue between
+          multiple parties, check out the{' '}
+          <Link to='/prob-revshare'>Probabilistic Revshare Generator</Link>.
+        </p>
+        <div className='tag-output'>
           <p>
-            This Link Tag Generator helps you generate your HTML link tag to
-            monetize your website.
-            <br />
-            Just provide your{' '}
-            <Link to='https://docs.openpayments.guide/docs/payment-pointers'>
-              Payment Pointer
-            </Link>{' '}
-            and click generate.
+            To monetize your website add the following <code>&lt;meta&gt;</code>{' '}
+            tag to the <code>&lt;head&gt;</code> section of all pages on your
+            website.
           </p>
-          <form id='paymentPointerForm' onSubmit={(ev) => ev.preventDefault()}>
-            <TextField
-              id='paymentPointer'
-              label='$YourPaymentPointer'
-              variant='outlined'
-              value={pointerInput}
-              error={Boolean(invalidUrl)}
-              helperText={
-                invalidUrl
-                  ? 'Please check the format of your payment pointer'
-                  : ''
-              }
-              fullWidth
-              onChange={(ev) => {
-                setPointerInput(ev.target.value)
-                setInvalidUrl('')
-              }}
-            />
-            <button
-              id='generateButton'
-              onClick={() => {
-                setInvalidUrl(false)
-                setPointer(parsePointerUrl(pointerInput))
-              }}
-            >
-              Generate
-            </button>
-          </form>
-          <p>
-            Read our <Link to='/docs'>docs</Link> to learn more about Web
-            Monetization. If you're interested in splitting revenue between
-            multiple parties, check out the{' '}
-            <Link to='/prob-revshare'>Probabilistic Revshare Generator</Link>.
-          </p>
-          <div className='metaTagOutput'>
-            <p>
-              To monetize your website add the following &lt;meta&gt; tag to the
-              &lt;head&gt; section of all pages on your website.
-            </p>
-            <code id='metaTag'>
-              &lt;link rel="monetization" href="{pointer}" /&gt;
-              <img
-                src='/img/copy_icon.svg'
-                id='copyIcon'
-                className='btnClipboard'
-                alt='copy-icon'
-              />
-            </code>
-          </div>
-        </Container>
-      </div>
+          <CodeBlock className='language-html'>
```

Comment:
Docusaurus does allow for use of the built-in default code syntax highlighter in custom pages as well, and we should make use of this instead of rolling our own.
See https://github.com/facebook/docusaurus/issues/6185